packages/mask/src/setup.ui.1.ts

```diff
@@ -0,0 +1,32 @@
+import Services from './extension/service'
```

Comment:
why your PR contains this file? it hasn't been merged to the main branch yet...

Reply:
Only for test